hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/server/TestSCMContainerMetrics.java

```diff
@@ -78,7 +78,9 @@ public void testSCMContainerMetrics() {
         "Number of containers in deleting state"), 6);
     verify(mb, times(1)).addGauge(Interns.info("DeletedContainers",
         "Number of containers in deleted state"), 7);
+    verify(mb, times(1)).addGauge(Interns.info("UnknownContainers",
```

Comment:
Could this cause the **basic (unit)** [failure](https://github.com/apache/ozone/actions/runs/3565038544/jobs/5989724741)? I think the `"UnknownContainers"` should be added [here](https://github.com/apache/ozone/blob/master/hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMContainerMetrics.java#L75L96).

Reply:
@dombizita - This will not fail, as I have take care not to add Unknown state as part of metrics, so no need to add above line.